providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/e2e/steps/config/ConfigSteps.java

```diff
@@ -0,0 +1,103 @@
+package dev.openfeature.contrib.providers.flagd.e2e.steps.config;
+
+import dev.openfeature.contrib.providers.flagd.Config;
+import dev.openfeature.contrib.providers.flagd.FlagdOptions;
+import dev.openfeature.contrib.providers.flagd.resolver.grpc.cache.CacheType;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ConfigSteps {
+
+    FlagdOptions.FlagdOptionsBuilder builder = FlagdOptions.builder();
+    FlagdOptions options;
+
+    @When("we initialize a config")
```

Comment:
This is a bit of a nit, but IMO we should use passive voice for these ... so instead of:

`we initialize a config`

It should be:

`a config is initialized`

Reply:
I will adapt the gherkin file, to be more precise

Reply:
... but as this is also already in a python pr, I would love to tackle our whole Gherkin files as one  - I think we can improve them and how we're using them. Currently it is quiet hard in all our SDKs repos, with no separations between steps etc